Makefile.core.mk

```diff
@@ -23,6 +23,7 @@ VERSION ?= 1.1.0
 MINOR_VERSION := $(shell echo "${VERSION}" | cut -f1,2 -d'.')
 
 OPERATOR_NAME ?= sailoperator
+VERSIONS_YAML_DIR = pkg/istioversions
 VERSIONS_YAML_FILE ?= versions.yaml
```

Comment:
Any reason why we use `=`  for `VERSIONS_YAML_DIR`, but `?=`  for `VERSIONS_YAML_FILE`?

Reply:
the directory is not subject to change in a vendored product, it's fixed. It's just in an env var because it's used in multiple places. The file name however needs to be adjusted in a vendored product

Reply:
Yeah, but it can still be changed via an env var, so might as well use consistent syntax.